server/routes/controllers/usersController.js

```diff
@@ -92,37 +92,18 @@ const usersController = {
   },
 
   changePassword (req, res) {
-    const users = this.Users
     const {email, password} = req.body
 
-    users.update({'email': email},
-      {$set: {'password': password}}, (error, r) => {
-        if (error) {
-          console.error(error)
-          return res.sendStatus(500)
-        }
-        if (r) {
-          if (r.n === 0) {
-            console.error(`Email doesn't exist`)
-            res.statusMessage = `Email doesn't exist`
-            return res.sendStatus(500)
-          } else {
-            if (r.nModified === 1) {
-              console.log(`New password for ${email} saved successfully!`)
-            }
-            users.findOne({ email }, (err, user) => {
-              if (err) {
-                console.error(err)
-                return res.sendStatus(500)
-              }
-
-              if (user) {
-                return res.send(user.toJSON())
-              }
-            })
-          }
-        }
-      })
+    var query = {'email': email}
```

Comment:
Don't see where this value changes, can be `const`

Reply:
damn copy paste :P eheh